### PR TITLE
Feature/export wizard

### PR DESCRIPTION
**Pull request by kevgliss**
Initial work on an export wizard. Some pieces will need to be refactored if we extend this pattern to other data models. 

Screenshot from 2020-11-30 10-24-50
Screenshot from 2020-11-30 10-24-32
Screenshot from 2020-11-30 10-24-13

### CODE REVIEW

**Comment by forestmonster** [comment]
Nice to clearly see where you are in the multi-step process. I also like the indicator showing the number of rows to be exported.

File: src/dispatch/static/dispatch/src/incident/TableExportDialog.vue [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+              <v-list-item>
+                <v-list-item-content>
+                  <incident-priority-combobox v-model="filters.incident_priority" />
+                </v-list-item-content>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-content>
+                  <incident-status-multi-select v-model="filters.status" />
+                </v-list-item-content>
+              </v-list-item>
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/incident/TableExportDialog.vue:50]: Are we missing a label for these three?

File: src/dispatch/static/dispatch/src/incident/TableExportDialog.vue [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+import IncidentStatusMultiSelect from "@/incident/IncidentStatusMultiSelect.vue"
+import TagFilterCombobox from "@/tag/TagFilterCombobox.vue"
+import IncidentTypeCombobox from "@/incident_type/IncidentTypeCombobox.vue"
+import IncidentPriorityCombobox from "@/incident_priority/IncidentPriorityCombobox.vue"
+export default {
+  name: "IncidentTableExportDialog",
+  data() {
+    return {
+      e1: 1,
+      selectedFields: [
```
Comment (mvilanova) [inline src/dispatch/static/dispatch/src/incident/TableExportDialog.vue:117]: Should type and perhaps priority be also selected by default?

**Review by mvilanova — APPROVED** [review]
Echoing @forestmonster feedback. Nice work!

File: src/dispatch/static/dispatch/src/incident/TableExportDialog.vue (hunk 1 above)
Comment (kevgliss) [reply src/dispatch/static/dispatch/src/incident/TableExportDialog.vue:50]: I believe they all have default labels within the components themselves.

**Pull request merged**